core/src/test/java/ai/timefold/solver/core/api/solver/SolverFactoryTest.java

```diff
@@ -237,8 +237,7 @@ void validateInitializationPhases() {
         solverConfig = PlannerTestUtils
                 .buildSolverConfig(TestdataSolution.class, TestdataEntity.class)
                 .withPhases(new CustomPhaseConfig()
-                        .withCustomPhaseCommandList(List.of(scoreDirector -> {
-                        })),
+                        .withCustomPhaseCommands(new NoChangeCustomPhaseCommand()),
```

Comment:
We should consider moving the `NoChange*` related models to the test package. The main issue would be the duplication.